### ISSUE TITLE
Schema Manager doesn't always show classes in the correct category (Vertex/Edge/Generic)

### ISSUE THREAD

**Issue by SamHard** [orientechnologies/orientdb-studio#460]
For classes with more than one superclass, the Schema Manager may not show classes in the correct category (Vertex/Edge/Generic).  It appears that only the first superclass is examined by the sorting algorithm.

Example scenario:
This script creates three classes that extend V.
```
CREATE CLASS ExampleA EXTENDS V;
CREATE CLASS ExampleB EXTENDS ORestricted, ExampleA;
CREATE CLASS ExampleC EXTENDS ExampleA, ORestricted;
```
Expected behavior is that all three appear in the Vertex Classes category in Schema Manager.
Actual behavior is that only ExampleA and ExampleC are displayed as Vertex Classes.  ExampleB appears under Generic Classes.

For the solution, you may want to look at this recursive JavaScript function, DoesClassHaveSuperClass(), in StackOverflow: http://stackoverflow.com/a/40183375/2832598

**Comment by maggiolo00** [comment]
Hi @SamHard 

Thanks for this issue. I guess the algorithm rely on the old field `superClass` instead of `superClasses` for multiple inheritance.

**Comment by maggiolo00** [comment]
hi @SamHard 

I've pushed a fix in 2.2.x and develop. It should be available in 2.2.18

Thanks

**Comment by SamHard** [comment]
That is great; thanks!
